app/src/main/java/com/sun/heartrate/ui/heartbeat/HeartbeatContract.kt

```diff
@@ -6,6 +6,8 @@ interface HeartbeatContract {
     interface View {
         fun getCameraManager(): CameraManager
         fun closeCamera()
+        fun updateUiTextViewHeart(rateNumber: Int)
```

Comment:
```suggestion
        fun displayHeatRate(rateNumber: Int)
```
em nên đặt tên function có ý nghĩa hơn 1 chút